site/docs/v1/tech/apis/users.adoc

```diff
@@ -490,6 +490,14 @@ Password hashes can be imported into FusionAuth. This allows users to transparen
 
 You can also learn more about hashes in the link:/docs/v1/tech/reference/password-hashes[password hashing reference], and you can link:/docs/v1/tech/plugins/custom-password-hashing[implement your own custom hash] as well.
 
+=== Encoding
+
+Unless you are using a custom plugin that does not work with base64 encoding, the password hash should be base64 encoded.
```

Comment:
In other words, base64 is the default encoding for the password hash.  If your password hashes are encoded in a different format, they will need to be imported as base64 or a custom plugin will be required (which an then make use of an alternative encoding). 


Maybe?   to make is even more apparent?   Might be too much duplication :) DRY....